aten/src/THC/THCBlas.cu

```diff
@@ -286,10 +286,19 @@ void THCudaBlas_Hgemm(THCState *state, char transa, char transb, int64_t m, int6
       cublasSetStream(handle, THCState_getCurrentStream(state));
 
 #ifdef __HIP_PLATFORM_HCC__
+#if __hcc_workweek__ < 18451
```

Comment:
I suggest let's only support single version of ROCm for now as the whole platform is moving quickly.

Reply:
Fair enough, will clean this out.